OpenRA.Mods.Common/Lint/CheckTooltips.cs

```diff
@@ -26,7 +26,7 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Ruleset ru
 				if (buildable == null)
 					continue;
 
-				var tooltip = actorInfo.Value.TraitInfos<TooltipInfo>().FirstOrDefault(Exts.IsTraitEnabled);
+				var tooltip = actorInfo.Value.TraitInfos<TooltipInfo>().FirstEnabledTraitOrDefault();
```

Comment:
By the way, does `Exts.IsTraitEnabled` work on `TraitInfo`s? I figured it was just for traits (hoping to learn something :))

Reply:
It doesn't. This is a copy-pasta error of an older logical error.  Lets get rid of it.

Reply:
You can use `ConditionalTraitInfo.EnabledByDefault` instead.

Reply:
A cursory glance shows places other than just tooltips affected, I have created #14445 to track this instead.

Reply:
Wouldn't it still be better to fix the places that are touched by this PR, though?

The old bogus look-up would just ignore the Exts.IsTraitsEnabled, but now this uses trait-specific look-up for TraitInfo, will this even work (without causing regressions vs. bleed, I mean)?

Besides, I couldn't find any other places (or rather, the only other similar place I could find already uses EnabledByDefault as @pchote suggested), though it's possible my search approach didn't catch them all.